samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/delivery/DeliveryBean.java

```diff
@@ -1859,41 +1911,45 @@ public String next_page()
     forGrade = false;
 
     if (this.actionMode == TAKE_ASSESSMENT
-        || this.actionMode == TAKE_ASSESSMENT_VIA_URL)
+            || this.actionMode == TAKE_ASSESSMENT_VIA_URL)
     {
       syncTimeElapsedWithServer();
-	
+
       SubmitToGradingActionListener listener =
-        new SubmitToGradingActionListener();
+              new SubmitToGradingActionListener();
       try {
-    	  listener.processAction(null);
+        listener.processAction(null);
       }
       catch (FinFormatException e) {
-		  log.debug(e.getMessage());
-		  return "takeAssessment";
-	  }
-	  catch (SaLengthException sae) {
-		  log.debug(sae.getMessage());
-		  return "takeAssessment";
-	  }
+        log.debug(e.getMessage());
+        return "takeAssessment";
+      }
+      catch (SaLengthException sae) {
+        log.debug(sae.getMessage());
+        return "takeAssessment";
+      }
     }
```

Comment:
All the above looks like just whitespace changes. I know it's tempting, my OCD wants to do this all the time. Unfortunately it's a history killer and we need to avoid doing this.